cmd/clusterctl/cmd/alpha_phase_get_kubeconfig.go

```diff
@@ -88,7 +88,7 @@ func init() {
 	alphaPhaseGetKubeconfigCmd.Flags().StringVarP(&pgko.Kubeconfig, "kubeconfig", "", "", "Path for the kubeconfig file to use")
 	alphaPhaseGetKubeconfigCmd.Flags().StringVarP(&pgko.ClusterName, "cluster-name", "", "", "Cluster Name")
 	// TODO: Remove as soon as code allows https://github.com/kubernetes-sigs/cluster-api/issues/157
-	alphaPhaseGetKubeconfigCmd.Flags().StringVarP(&pgko.Provider, "provider", "", "", "Which provider deployment logic to use (google/vsphere/azure)")
+	alphaPhaseGetKubeconfigCmd.Flags().StringVarP(&pgko.Provider, "provider", "", "", "Which provider deployment logic to use (google/vsphere/azure/openstack/vsphere/baidu/tencent/digitalocean/openshift)")
```

Comment:
I wonder if instead of adding more providers to the description here if we should just remove the provider names all together, especially since you need a provider-provided `clusterctl` binary at this time.

Reply:
This make sense, thanks @detiber , PR updated.